### PR TITLE
arreglo mayusculas

### PR DESCRIPTION
**Pull request by SebasFelix**

### CODE REVIEW

File: spec/controllers/game_controller_spec.js [hunk 1]
```diff
@@ -451,7 +451,7 @@ describe('GameController', () => {
         request(app).post('/games')
           .send(game_7)
           .end((err, res) => {
-            expect(res.body.error).to.eq('RANKING MUST HAVE A POINTS');
+            expect(res.body.error).to.eq('Ranking must have a points');
```
Comment (guilleijo) [inline spec/controllers/game_controller_spec.js]: Sacale el 'a', 'Ranking must have points'

File: spec/controllers/game_controller_spec.js [hunk 2]
```diff
@@ -505,7 +505,7 @@ describe('GameController', () => {
         request(app).post('/games')
           .send(game_13)
           .end((err, res) => {
-            expect(res.body.error).to.eq('THERE MUST WRITE BETWEEN 2 AND 6 POSSIBLES ANSWERS');
+            expect(res.body.error).to.eq('There must write between 2 and 6 possibles answers');
```
Comment (guilleijo) [inline spec/controllers/game_controller_spec.js]: 'The question must have between 2 and 6 possibles answers'

File: spec/controllers/game_controller_spec.js [hunk 3]
```diff
@@ -523,7 +523,7 @@ describe('GameController', () => {
         request(app).post('/games')
           .send(game_15)
           .end((err, res) => {
-            expect(res.body.error).to.eq('YOU MUST SELECT CORRECT ANSWER');
+            expect(res.body.error).to.eq('You must select correct answer');
```
Comment (guilleijo) [inline spec/controllers/game_controller_spec.js]: 'You must select a correct answer'

File: spec/controllers/game_controller_spec.js [hunk 4]
```diff
@@ -532,7 +532,7 @@ describe('GameController', () => {
         request(app).post('/games')
           .send(game_16)
           .end((err, res) => {
-            expect(res.body.error).to.eq('THERE MUST BE A CORRECT ANSWER');
+            expect(res.body.error).to.eq('There must be a correct answer');
```
Comment (guilleijo) [inline spec/controllers/game_controller_spec.js]: No seria igual al mensaje de arriba? Si es el mismo error que diga lo mismo.

File: spec/controllers/matches_controller_spec.js [hunk 5]
```diff
@@ -357,7 +357,7 @@ describe('MatchesController', () => {
         request(app).post('/matches')
           .send(match_4)
           .end((err, res) => {
-            expect(res.body.error).to.eq('MUST HAVE A OWNER');
+            expect(res.body.error).to.eq('Must have a owner');
```
Comment (guilleijo) [inline spec/controllers/matches_controller_spec.js]: 'Must have an owner'

File: spec/controllers/matches_controller_spec.js [hunk 6]
```diff
@@ -375,7 +375,7 @@ describe('MatchesController', () => {
         request(app).post('/matches')
           .send(match_6)
           .end((err, res) => {
-            expect(res.body.error).to.eq('RESULT MUST HAVE A POINTS');
+            expect(res.body.error).to.eq('Result must have a points');
```
Comment (guilleijo) [inline spec/controllers/matches_controller_spec.js]: 'Result must have points'

File: spec/controllers/users_controller_spec.js [hunk 7]
```diff
@@ -287,7 +287,7 @@ describe('UsersController', () => {
         request(app).post('/users')
           .send(user_1)
           .end((err, res) => {
-            expect(res.body.error).to.eq('YOU MUST ENTER A EMAIL');
+            expect(res.body.error).to.eq('You must enter a email');
```
Comment (guilleijo) [inline spec/controllers/users_controller_spec.js]: ... an email'

File: spec/controllers/game_controller_spec.js (hunk 4 above)
Comment (SebasFelix) [reply spec/controllers/game_controller_spec.js]: uno es si no se define y el otro si se define mal

**Pull request merged**